libs/caseStatuses.js

```diff
@@ -8,6 +8,7 @@ import {
   CLOSED,
   NOT_STARTED_VIVA,
   ACTIVE_COMPLETION_REQUIRED_VIVA,
+  ACTIVE_COMPLETION_RANDOM_CHECK_REQUIRED_VIVA,
```

Comment:
Can we rename this to `ACTIVE_RANDOM_CHECK_REQUIRED_VIVA`? The current name feels misleading, as `COMPLETION` is saying we need to send in additional details.

Reply:
I agree with @Sven65. 
A "normal" stickprovskontroll should only have the name:
`ACTIVE_RANDOM_CHECK_REQUIRED_VIVA`

which means that if we want a user to send in `completions` for a stickprovskontroll then we could add:
`ACTIVE_COMPLETION_RANDOM_CHECK_REQUIRED_VIVA`

What do you think about that @dannil76?

Reply:
will rename to ACTIVE_RANDOM_CHECK_REQUIRED_VIVA and keep ACTIVE_COMPLETION_REQUIRED_VIVA as is